lib/core/lib/ops/userOps.js

```diff
@@ -20,6 +21,49 @@ function buildUserOps (db/*, config */) {
     job.client.query(sqlQuery, utils.boomErrorWrapper(next))
   }
 
+  function readUserTeams (job, next) {
+    let { id, organizationId, limit, page } = job
+    let sqlQuery = SQL`
+        SELECT
+          teams.id,
+          teams.name,
+          teams.description,
+          teams.path,
```

Comment:
Pretty sure there is no point in returning `org_id` as you're having it as a param. Most likely `path` is not used as well.